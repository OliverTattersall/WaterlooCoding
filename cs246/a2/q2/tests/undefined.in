create a +infinity
create b -infinity
create c 0
create d undefined
* a b
* a a
- a a 
- a b 
- b b 
- b a 
* a c
* b c 
/ c c 
/ a b
/ a d
u - d
print a 
print b 
print c 
print d 
