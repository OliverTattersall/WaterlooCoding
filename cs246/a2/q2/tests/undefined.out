-infinity
+infinity
undefined
+infinity
undefined
-infinity
undefined
undefined
undefined
undefined
undefined
undefined
-infinity
0
undefined
